org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -407,6 +464,26 @@ package lang {
 		override method asSet() = self
 
 		override method anyOne() native
+
+		/**
+		 * Returns a new Set with the elements of both self and another collection.
+		 * @returns a Set
+		 */
+		 method union(another) = self + another
```

Comment:
I'm sorry, I must have messed something up because I don't see it in my branch. I don't really understand how it came to this, I have it locally in my PC because it was implemented in the wdate branch recently, but it doesn't show up in here, not even as deleted.

Now I'll have to review everything to check if it's the only thing missing :(